src/bric.c

```diff
@@ -1579,7 +1582,7 @@ int handle_tag_movement(int where) {
 	strcpy(orig_filename, Editor.filename);
 	if(where == MOVE_BACK) {
 		if(isempty(&tag_stack)) {
-			editor_set_status_message("at bottom of tag stack");
+			editor_set_status_message(_("at bottom of tag stack"));
```

Comment:
This string my be ambiguous to translators. https://www.gnu.org/software/gettext/manual/html_node/Preparing-Strings.html#Preparing-Strings

I assume "you are" is implied, but in some languages, grammar changes depending on the subject, so I suggest leaving a TRANSLATOR note here or adjusting the string.